apps/client/.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "singleQuote": true,
```

Comment:
Why change the quoting?

Reply:
Prefer to always use single quotes, except when we use double quotes to avoid escaping a single quote.
Since this is a personal preference limited this to only the client application.

Reply:
We should have our formatting settings spread across the monorepo. Ideally, the two environments should only slightly vary where it's required.

I prefer to use double, since it is the standard string syntax across most programming languages, but I will support the team's preference.

Reply:
This is the preference of the team in the other packages in [@iot-app-kit](https://github.com/awslabs/iot-app-kit/)

Reply:
Cool. Let's change it then.